src/Cardano/Address/Derivation.hs

```diff
@@ -190,4 +366,14 @@ class GenMasterKey (key :: Depth -> * -> *) where
     type GenMasterKeyFrom key :: *
 
     -- | Generate a root key from a corresponding seed.
-    genMasterKey :: GenMasterKeyFrom key -> ScrubbedBytes -> key 'RootK XPrv
+    genMasterKey :: GenMasterKeyFrom key -> key 'RootK XPrv
+
+
+--
+-- Internals
+--
+
+-- Simple helper to convert 'Either e' to 'Maybe' until we have a slightly
+-- better approach for error.
+eitherToMaybe :: Either e a -> Maybe a
```

Comment:
why not to use `import Data.Either.Extra (eitherToMaybe)` we import already `extra` package

Reply:
done

Reply:
Ah! I didn't notice, but good point!